src/task-1/GenreSelect/GenreSelect.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component } from 'react';
+import './GenreSelect.css';
+
+type Genre = 'Crime' | 'Documentary' | 'Horror' | 'Comedy';
+
+const GenreCheckbox = (props: { genre: Genre }) => (
+  <div className='checkbox'>
+    <input type='checkbox' />
+    <label>{props.genre}</label>
+  </div>
+);
+
+interface IProps {}
+interface State {
+  isOpen: boolean;
+}
+
+export class GenreSelect extends Component<IProps, State> {
```

Comment:
Do you need this iprops repeated ? perpahs anonymous interface could be used?

Reply:
fixed